<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
<!--
SPDX-FileCopyrightText: 2024 Carl Schwan <carl@carlschwan.eu>
SPDX-License-Identifier: LGPL-2.0-or-later
-->
 <class>ParagraphDecorations</class>
 <widget class="QWidget" name="ParagraphDecorations" >
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="0">
      <widget class="QLabel" name="backgroundColorLabel" >
       <property name="text" >
        <string>Background color:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>backgroundColor</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item column="0">
        <widget class="KColorButton" native="1" name="backgroundColor" />
       </item>
       <item column="1">
        <widget class="QToolButton" name="resetBackgroundColor" >
         <property name="icon">
          <iconset theme="edit-reset-symbolic"/>
         </property>
         <property name="toolTip">
          <string>Reset background color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QWidget</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
</ui>
